src/actinia_module_plugin/endpoints.py

```diff
@@ -45,6 +45,36 @@
 from actinia_module_plugin.api.actinia_templates import ActiniaTemplate
 from actinia_module_plugin.api.actinia_templates import ActiniaTemplateId
 
+from actinia_core.endpoints import get_endpoint_class_name
```

Comment:
Isn't this a circular reference? Does ist start like this?

Reply:
Yes it started.

Reply:
Yes it did